client-js/app/scripts/places.ts

```diff
@@ -0,0 +1,29 @@
+// Function for creating URLs of all the places in the application.
+
```

Comment:
That's correct. The "routing" happens in `app.ts` which uses the page.js library to set up routes that launch various activities. We should probably refactor this to keep the info about creating and parsing urls in this one module, but that will be for another PR.